src/pages/iou/request/MoneyRequestParticipantsSelector.tsx

```diff
@@ -105,28 +105,31 @@ function MoneyRequestParticipantsSelector({
             };
         }
 
-        const optionList = OptionsListUtils.getFilteredOptions({
-            reports: options.reports,
-            personalDetails: options.personalDetails,
-            betas,
-            selectedOptions: participants as Participant[],
-            excludeLogins: CONST.EXPENSIFY_EMAILS,
-
-            // If we are using this component in the "Submit expense" or the combined submit/track flow then we pass the includeOwnedWorkspaceChats argument so that the current user
-            // sees the option to submit an expense from their admin on their own Workspace Chat.
-            includeOwnedWorkspaceChats: (iouType === CONST.IOU.TYPE.SUBMIT || iouType === CONST.IOU.TYPE.CREATE || iouType === CONST.IOU.TYPE.SPLIT) && action !== CONST.IOU.ACTION.SUBMIT,
-
-            includeP2P: !isCategorizeOrShareAction,
-            canInviteUser: !isCategorizeOrShareAction,
-            includeInvoiceRooms: iouType === CONST.IOU.TYPE.INVOICE,
-            action,
-            sortByReportTypeInSearch: isPaidGroupPolicy,
-            searchValue: '',
-            maxRecentReportsToShow: 0,
-        });
+        const optionList = OptionsListUtils.getOptions(
+            {
+                reports: options.reports,
+                personalDetails: options.personalDetails,
+            },
+            {
+                betas,
+                selectedOptions: participants as Participant[],
+                excludeLogins: CONST.EXPENSIFY_EMAILS,
+
+                // If we are using this component in the "Submit expense" or the combined submit/track flow then we pass the includeOwnedWorkspaceChats argument so that the current user
+                // sees the option to submit an expense from their admin on their own Workspace Chat.
+                includeOwnedWorkspaceChats:
+                    (iouType === CONST.IOU.TYPE.SUBMIT || iouType === CONST.IOU.TYPE.CREATE || iouType === CONST.IOU.TYPE.SPLIT) && action !== CONST.IOU.ACTION.SUBMIT,
+
+                includeP2P: !isCategorizeOrShareAction,
+                canInviteUser: !isCategorizeOrShareAction,
+                includeInvoiceRooms: iouType === CONST.IOU.TYPE.INVOICE,
+                action,
+                maxRecentReportsToShow: 0,
+            },
+        );
 
         return optionList;
-    }, [action, areOptionsInitialized, betas, didScreenTransitionEnd, iouType, isCategorizeOrShareAction, options.personalDetails, options.reports, participants, isPaidGroupPolicy]);
+    }, [action, areOptionsInitialized, betas, didScreenTransitionEnd, iouType, isCategorizeOrShareAction, options.personalDetails, options.reports, participants]);
 
```

Comment:
I think we may miss the sorting by type here after removing `sortByReportTypeInSearch: isPaidGroupPolicy`

I noticed ordering change between this PR and staging


https://github.com/user-attachments/assets/3574f10b-6fad-421c-b96d-e5bd85f9439d

Reply:
okay, addressed the ordering as well. 

Note: there are now two places where we explicitly call `.orderOptions`. In the next PR i will finally separate the filtering and ordering in a clean manner. So this redundancy will go away in the next PR